edge-functions/create-react-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
```

Comment:
Wdyt about adding another paragraph saying "Open devtools and you should be able to see `x-cra-example` in the list, added by middleware.js".